install/installer/pkg/components/ws-manager-mk2/configmap.go

```diff
@@ -221,10 +221,11 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 				Interrupted:         util.Duration(5 * time.Minute),
 			},
 			//EventTraceLog:                "", // todo(sje): make conditional based on config
-			ReconnectionInterval:           util.Duration(30 * time.Second),
-			RegistryFacadeHost:             fmt.Sprintf("reg.%s:%d", ctx.Config.Domain, common.RegistryFacadeServicePort),
-			WorkspaceCACertSecret:          customCASecret,
-			TimeoutMaxConcurrentReconciles: 5,
+			ReconnectionInterval:             util.Duration(30 * time.Second),
+			RegistryFacadeHost:               fmt.Sprintf("reg.%s:%d", ctx.Config.Domain, common.RegistryFacadeServicePort),
+			WorkspaceCACertSecret:            customCASecret,
+			WorkspaceMaxConcurrentReconciles: 15,
+			TimeoutMaxConcurrentReconciles:   15,
```

Comment:
set both the timeout and workspace controller's max reconciles to 15. This number is slightly arbitrary but should be sufficient for us looking at the metrics during the loadgen. It's in config, so we can easily change it anyways